otc_sphinx_directives/tests/test_directive_wrapper.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2010-2011 OpenStack Foundation
+# Copyright (c) 2013 Hewlett-Packard Development Company, L.P.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+from bs4 import BeautifulSoup
+
+from otc_sphinx_directives.tests import base
+
+
+def get_ref_content(wrapper_type=None):
+    ref_content = ()
+
+    if wrapper_type:
+        ref_content += (f'<{wrapper_type} class="ecs" id="7891011">',)
+    else:
+        ref_content += ('<div class="ecs" id="123456">',)
+
+    ref_content += (
+        '<div class="admonition note">',
```

Comment:
are you sure you want to deal with tuple elements and not a single string?

Reply:
this tuple is still not address

Reply:
The tuple solution is much more elegant for multiple lines. With string you have to strip out \n and spaces which always consumes efficiency.

Reply:
done